dts/common/nordic/nrf54h20.dtsi

```diff
@@ -68,6 +68,56 @@
 				nordic,tasks-mask = <0xfffffff0>;
 			};
 		};
+
+		cpuflpr: cpu@e {
+			compatible = "nordic,vpr";
+			reg = <14>;
+			device_type = "cpu";
+			clock-frequency = <DT_FREQ_M(320)>;
+			riscv,isa = "rv32emc";
+			nordic,bus-width = <64>;
+
+			cpuflpr_vevif_rx: mailbox {
+				compatible = "nordic,nrf-vevif-task-rx";
+				status = "disabled";
+				interrupt-parent = <&cpuflpr_clic>;
+				interrupts = <0 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <1 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <2 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <3 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <4 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <5 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <6 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <7 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <8 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <9 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <10 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <11 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <12 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <13 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <14 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <15 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <16 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <17 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <18 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <19 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <20 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <21 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <22 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <23 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <24 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <25 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <26 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <27 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <28 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <29 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <30 NRF_DEFAULT_IRQ_PRIORITY>,
+					     <31 NRF_DEFAULT_IRQ_PRIORITY>;
+				#mbox-cells = <1>;
+				nordic,tasks = <32>;
```

Comment:
This should be 16.

Reply:
We need to review all this, I agree it's confusing to have some numbers that are later not allowed by the mask. We need some guidance from @hubertmis here (docs are not so clear).

Reply:
ref 

![image](https://github.com/user-attachments/assets/958b5a34-ac5a-437c-b328-ad47dc7930e3)

Reply:
I'm too late with the guidance I received from VPR experts. It seems "VEVIF tasks:" should be "16..31", what would translate `<nordic,tasks = <16>;`

Reply:
just checked: code in tasks drivers is index sensitive, as channel number is given in the 0...31 range and it uses this index to pick from an indexed-array. So for now, I'd live driver and DT as is, from a user perspective, nothing changes really. To improve this, we'd need additional information like, do we expect non-contiguous holes in the mask?

Reply:
> we'd need additional information like, do we expect non-contiguous holes in the mask?

I believe so. I think I've seen VPR instances with non-contiguous events or tasks. Those could have been experimental though. I don't remember the details.

Anyway, I agree there is not rush in improving this.